plopfile.mjs

```diff
@@ -11,6 +11,11 @@ const DEFAULT_CONFIGS = {
     corpusId: "1",
     apiKey: "zqt_UXrBcnI2UXINZkrv4g1tQPhzj02vfdtqYJIDiA",
     appTitle: "Vectara Docs Search",
+    search_title: "Ask Vectara Docs",
+    search_description: "Ask anything about Vectara Docs",
+    summary_default_language: "eng",
```

Comment:
summary_default_language, summary_num_sentences, and summary_num_results are already part of the env generation template, so we don't need them here. :+1:

Reply:
What values do they take?